src/components/background/styles.js

```diff
@@ -6,17 +6,16 @@ const { getColorClassName } = wp.blockEditor;
 /**
  * Background Classes
  */
-function BackgroundStyles( attributes ) {
-
-	const backgroundClass = getColorClassName( 'background-color', attributes.backgroundColor );
+const BackgroundStyles = ( attributes, backgroundColor ) => {
+	const backgroundClass = attributes && getColorClassName( 'background-color', attributes.backgroundColor );
 
 	const styles = {
-		backgroundImage: attributes.backgroundImg && attributes.backgroundType == 'image' ? `url(${ attributes.backgroundImg })` : undefined,
-		backgroundColor: backgroundClass ? undefined : attributes.customBackgroundColor,
-		backgroundPosition:  attributes.focalPoint && !  attributes.hasParallax ? `${  attributes.focalPoint.x * 100 }% ${  attributes.focalPoint.y * 100 }%` : undefined,
+		backgroundImage: attributes.backgroundImg && attributes.backgroundType === 'image' ? `url(${ attributes.backgroundImg })` : undefined,
+		backgroundColor: backgroundClass ? backgroundColor && backgroundColor.color : attributes.customBackgroundColor,
```

Comment:
Optional chaining might be used here to avoid short-circuit evaluation: https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining